cluster/test-deploy/gcp-dev/vars-origin.yaml

```diff
@@ -0,0 +1,124 @@
+
+container_runtime_docker_storage_type: overlay2
+
+provision_custom_repositories: [
+  {
+    "name": "oso-rhui-rhel-server-releases",
+    "baseurl": "https://mirror.ops.rhcloud.com/libra/rhui-rhel-server-7-releases/
+        https://use-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-releases/
+        https://use-mirror2.ops.rhcloud.com/libra/rhui-rhel-server-7-releases/
+        https://euw-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-releases/
+        https://gce-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-releases/
+        https://gce-mirror2.ops.rhcloud.com/libra/rhui-rhel-server-7-releases/",
+    "sslverify": "false",
+    "sslclientkey": "ops-mirror.pem",
+    "sslclientcert": "ops-mirror.pem",
+    "failovermethod": "priority",
+    "gpgcheck": 1,
+    "gpgkey": "https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted"
+  },
+  {
+    "name": "oso-rhui-rhel-server-extras",
+    "baseurl": "https://mirror.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://use-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://use-mirror2.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://euw-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://gce-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://gce-mirror2.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/",
+    "sslverify": "false",
+    "sslclientkey": "ops-mirror.pem",
+    "sslclientcert": "ops-mirror.pem",
+    "failovermethod": "priority",
+    "gpgcheck": 1,
+    "gpgkey": "https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted"
+  },
+  {
+    "name": "dockertested",
+    "baseurl": "https://mirror.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://use-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://use-mirror2.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://euw-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://gce-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://gce-mirror2.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/",
+    "sslverify": "false",
+    "sslclientkey": "ops-mirror.pem",
+    "sslclientcert": "ops-mirror.pem",
+    "failovermethod": "priority",
+    "gpgcheck": 1,
+    "gpgkey": "https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted"
+  }
+]
+
+openshift_gcp_prefix: "ci-{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}-"
```

Comment:
if INSTANCE_PREFIX isn't set this will be `ci--`

Reply:
isn't INSTANCE_PREFIX implicitly required in the Makefile via WHAT, and then explicitly required here via the `mandatory` filter?